scripts/npc/2042005.js

```diff
@@ -45,15 +45,17 @@ function action(mode, type, selection) {
 
                 if (party >= 1) {
                     status = 10;
-                    cm.sendOk("Voc� n�o tem n�mero suficiente de pessoas em seu grupo. Voc� precisa de um grupo com #b" + cpqMinAmt + "#k - #r" + cpqMaxAmt + "#k membros e eles devem estar no mapa com voc�.");
+                    cm.sendOk("You do not have enough people in your party. You need a party with #b" + cpqMinAmt + "#k - #r" + cpqMaxAmt + "#k members and they should be on the map with you.");
                 } else if (lvlOk != inMap) {
                     status = 10;
-                    cm.sendOk("Certifique se todos em seu grupo est�o dentre os n�veis corretos (" + cpqMinLvl + "~" + cpqMaxLvl + ")!");
+                    cm.sendOk("Make sure everyone in your party is among the correct levels (" + cpqMinLvl + "~" + cpqMaxLvl + ")!");
                 } else if (isOutMap > 0) {
                     status = 10;
-                    cm.sendOk("Existe algu�m do grupo que n�o esta no mapa!");
+                    cm.sendOk("There are some of the party members that is not on the map!");
                 } else {
-                    cm.sendCPQMapLists2();
+                    if (!cm.sendCPQMapLists2()) {
```

Comment:
What is this condition checking for? `sendCPQMapLists2` always returns `void`

Reply:
There is also language referenced in `NPCConversationManager.java` that uses `LanguageConstants.java` to write the final message. We need to ensure that the final message is also using the correct localization.